dir.targets

```diff
@@ -8,11 +8,16 @@
   <Target Name="_RestoreBuildToolsWrapper" DependsOnTargets="_DisplayBuildInfo; _RestoreBuildTools" />
 
   <Target Name="_DisplayBuildInfo">
+      <!-- Cannot do both MonoBuild and NetCoreBuild -->
+      <Error Condition="'$(MonoBuild)' == 'true' and '$(NetCoreBuild)' == 'true'"
+             Text="Cannot do both Mono and NetCore build."
+      />
+
     <Message Importance="High" Text="Build '$(MSBuildProjectFile)' into binary directory '$(BaseOutputPathWithConfig)'" />
   </Target>
 
   <Target Name="_CopyCompilers"
-          AfterTargets="_RestoreBuildToolsWrapper"
+          AfterTargets="_RestoreBuildToolsWrapper1"
```

Comment:
This basically disabled the targets. I am not ready to completely remove it yet, just in case... But I'll add a comment so we don't forget to do it when we are ready (the compilers are copied into the output using runtime.project.json -- we don't need to do it here.